jfr-mappers/build.gradle.kts

```diff
@@ -13,6 +18,23 @@ tasks {
     }
 }
 
+dependencies {
+    "api"("com.newrelic.telemetry:telemetry-all:${newRelicTelemetryVersion}")
+    implementation("com.google.code.gson:gson:${gsonVersion}")
+}
+
+jpmsExtraModules {
+    module("gson-${gsonVersion}.jar", "com.google.code.gson", gsonVersion) {
+        exports("com.google.gson")
+    }
+    module("mockito-junit-jupiter-${mockitoVersion}.jar", "mockito.junit.jupiter", mockitoVersion) {
+        exports("com.google.gson")
+    }
+    module("objenesis-${objenesisVersion}.jar", "org.objenesis", objenesisVersion) {
+        exports("com.google.gson")
+    }
+}
```

Comment:
I believe `jpmsExtraModules` is coming from the `org.ysb33r.java.modulehelper` plugin, but its not clear to me what that plugin is doing. Can you elaborate on what happening here?